apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -37,6 +37,9 @@
 #define DEVPATH "/dev/ttyS2"
 #endif
 
+#define LOOPBACK_ON	1	//set the loopback mode for UART
```

Comment:
Please use public definitions instead of local definitions in the app. Please find my previous comment.